src/pages/explore/BiddersPage/index.tsx

```diff
@@ -1,27 +1,62 @@
 import type { Vec } from '@polkadot/types'
 import type { PalletSocietyBid } from '@polkadot/types/lookup'
-import { useEffect, useState } from 'react'
-import { Spinner } from 'react-bootstrap'
+import { useEffect, useState, useCallback } from 'react'
+import { Spinner, Row, Col, Alert } from 'react-bootstrap'
+import styled from 'styled-components'
+import { useAccount } from '../../../account/AccountContext'
 import { useKusama } from '../../../kusama'
 import { BiddersList } from './BiddersList'
+import { BidVouch } from './BidVouch'
 
 const BiddersPage = (): JSX.Element => {
   const { api } = useKusama()
+  const { activeAccount } = useAccount()
   const [bids, setBids] = useState<Vec<PalletSocietyBid> | []>([])
+  const [result, setResult] = useState(null)
+  const [showAlert, setShowAlert] = useState(true)
 
   const loading = !api?.query?.society
 
   useEffect(() => {
     if (!loading) {
-      api.query.society.bids().then((response: Vec<PalletSocietyBid>) => {
+      api.query.society.bids((response: Vec<PalletSocietyBid>) => {
```

Comment:
By removing `then` and instead passing an unsubscribe function, we are effectively subscribing to changes on the bid list.